clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -47,7 +47,7 @@
   // Version 15 is the same as version 14 (KIP-903).
   //
   // Version 16 adds the 'NodeEndpoints' field (KIP-951).
-  "validVersions": "0-16",
+  "validVersions": "0-17",
```

Comment:
We need to add a comment to mention why we need to bump the version.

Reply:
Fixed.